### PR TITLE
3 notice functions merged into 1 

### PR DESCRIPTION
**Pull request by ramico**

### CODE REVIEW

File: includes/class-wc-siftscience-html.php [hunk 1]
```diff
@@ -83,23 +83,37 @@ class="<?php echo esc_attr( $class ); ?>">
...
 		}
 
+		/**
+		 * This function is to cdisplay a notice so the user shoulr update their plugin
+		 *
+		 * @param string $settings_url the link to update plugin.
+		 */
+		public function disply_update_notice( $settings_url ) {
+			?>
+			<div class="notice notice-error is-dismissible">
```
Comment (ramico) [inline includes/class-wc-siftscience-html.php:111]: this notice should not be is-dismissible

**Review by ramico — COMMENTED** [review]
remove css class

File: includes/class-wc-siftscience-admin.php [hunk 2]
```diff
@@ -567,69 +567,39 @@ private function create_element( $type, $id, $label = '', $desc = '', $element_o
...
-			$enabled = get_option( WC_SiftScience_Options::SEND_STATS, 'not_set' );
+				if ( false === $is_valid_conf ) {
+					$settings_url = admin_url( 'admin.php?page=wc-settings&tab=siftsci' );
 
-			if ( 'not_set' !== $enabled ) {
-				return;
+					$this->html->disply_update_notice( $settings_url );
+				}
 			}
+			// Check to dispkay improve notice.
```
Comment (nabsul) [inline includes/class-wc-siftscience-admin.php:586]: dispkay --> display

File: includes/class-wc-siftscience-html.php (hunk 1 above)
Comment (nabsul) [reply includes/class-wc-siftscience-html.php:111]: Why? We can discuss later.

**Review by nabsul — APPROVED** [review]

**Pull request merged**